Gopkg.lock

```diff
@@ -746,6 +746,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "f5ff9662192d238efdb260226a886088d05da0924292833660466258fd80c8c0"
```

Comment:
what changed in gopkg.lock that requires a go-deps change? afaict this bump looks unnecessary?

Reply:
without this change the `go vet ./...` in ci failed:
https://travis-ci.org/runconduit/conduit/jobs/370317684

i suspect it may be changes to various imports, but no dependencies have been added/removed from the repo.

Reply:
found it by diff'ing the output of `bin/dep hash-inputs` in master and this branch, new dependency:
k8s.io/apimachinery/pkg/api/errors